common/components/ConfirmationModalTemplate/components/Body/components/Details.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { fromTokenBase, getDecimalFromEtherUnit, Wei, Nonce, Data } from 'libs/units';
+import Code from 'components/ui/Code';
+import './Details.scss';
+import BN from 'bn.js';
+import { rawUnitConversion } from 'utils/rawUnitConversion';
+
+interface Props {
+  sendValue: BN;
+  network: string;
+  provider: string;
+  gasPrice: any;
+  gasLimit: any;
+  nonce: any;
+  data: any;
+  chainId: any;
+  decimal: number;
+  symbol: string;
+}
+
+export const Details: React.SFC<Props> = ({
```

Comment:
There could be way less props here if a selector was used instead to derive all the correct data